### PR TITLE
feat(toggle): migrate to new archi

### PR DESCRIPTION
**Pull request by skhamvon**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#1200.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build#1204.0 ✘  
* 
  * build ✘

File: packages-new/components/toggle/src/components/osds-toggle/interfaces/attributes.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
-import { OdsComponentAttributes } from '../ods-component-attributes';
 import { OdsThemeColorIntent } from '@ovhcloud/ods-theming';
```
Comment (dpellier) [inline packages-new/components/toggle/src/components/osds-toggle/interfaces/attributes.ts:1]: type

**Review by dpellier — CHANGES_REQUESTED** [review]

File: packages-new/components/toggle/src/components/osds-toggle/interfaces/attributes.ts (hunk 1 above)
Comment (skhamvon) [reply packages-new/components/toggle/src/components/osds-toggle/interfaces/attributes.ts]: Fixed

**Review by dpellier — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-storybook#1246.0 ✘  
* 
  * build ✘

**Pull request merged**